lib/index.ts

```diff
@@ -191,29 +198,30 @@ interface ExposableOptions {
   readonly deployment: DeboredDeployment,
   readonly port: number,
   readonly selector: { [key: string]: string }
-  readonly useIngress: boolean
+  readonly ingressType: IngressType
 }
 
 class Exposable extends Construct {
   constructor(scope: Construct, name: string, opts: ExposableOptions) {
     super(scope, name);
 
```

Comment:
I would recommend to explicitly map from the enum to the service type string here:

```ts
function serviceTypeFromIngressType(ingressType: IngressType): string {
  switch (ingressType) {
    case IngressType.SERVICE_LOADBALANCER: return 'LoadBalancer';
    case IngressType.CLUSTER_IP: return 'ClusterIP';
    case IngressType.NGINX_INGRESS: return 'ClusterIP';
    default: throw new Error('unsupported ingress type');
  }
}
```

Reply:
Awesome. I'm updating the code with this.

Reply:
Addressed with this commit: https://github.com/toricls/cdk8s-debore/pull/3/commits/d4e0761b1f4c2bc16c0f569758ab9b85c98e0779